apps/site/src/app/(main)/guest-login/GuestLogin.tsx

```diff
@@ -7,6 +7,7 @@ async function GuestLogin({
 	searchParams,
 }: {
 	searchParams?: {
```

Comment:
Thought: I think Next.js will always provide `searchParams` to server components, so I don't think this needs to be marked as optional.